doc/maintaining/installing/index-es.rst

```diff
@@ -0,0 +1,62 @@
+***
+
+## Instalación de CKAN
+
+Antes de que pueda usar CKAN en su propia computadora, debe instalarlo.
+Hay tres formas de instalar CKAN:
+
+\#. Instalar desde un paquete de sistema operativo
+\#. Instalar desde el origen
+\#. Instalar desde Docker Compose
+
+CKAN 2.9 soporta Python 3.7 y Python 3.8 más 2.7. La próxima versión de CKAN
+solo admitirá Python 3.
+
+Se pueden encontrar consejos de implementación adicionales en nuestro wiki, como los recomendados
+`Hardware Requirements <https://github.com/ckan/ckan/wiki/Hardware-Requirements>`\_.
+
+La instalación desde el paquete es la forma más rápida y fácil de instalar CKAN, pero requiere
+Ubuntu 18.04 de 64 bits o Ubuntu 20.04 de 64 bits.
+
+**Debe instalar CKAN desde el paquete si**:
+
+*   Desea instalar CKAN en un servidor Ubuntu 18.04 o 20.04, de 64 bits, *y*
+*   Solo desea ejecutar un sitio web CKAN por servidor
+
+Véase :d oc:`install-from-package`.
```

Comment:
All `:doc:` directives seem to be broken

Reply:
pls mark all suggestions